src/main/kotlin/com/github/devcordde/devcordbot/commands/general/RankCommand.kt

```diff
@@ -75,22 +83,24 @@ class RankCommand : AbstractCommand() {
         ).queue()
     }
 
-    private fun buildProgressBar(current: Long, next: Long): String? {
+    private fun buildProgressBar(current: Long, next: Long): String {
         val stringBuilder = StringBuilder()
         val barProgress = (current.toDouble() / next * 20).toInt()
         stringBuilder.append("█".repeat(barProgress))
             .append("▒".repeat(20 - barProgress))
         return stringBuilder.toString()
     }
 
-    private inner class TopCommand : AbstractSubCommand(this) {
-        override val aliases: List<String> = listOf("top", "t", "leaderboard", "thebest")
-        override val displayName: String = "Top"
+    private inner class TopCommand : AbstractSubCommand.Command(this) {
+        override val name: String = "top"
         override val description: String = "Zeigt die 10 User mit dem höchsten Rang an."
-        override val usage: String = "[offset]"
+        override val options: List<CommandUpdateAction.OptionData> = buildOptions {
+            int("offset", "Der Index um den die Liste verschoben werden soll") {
+            }
```

Comment:
I feel like this should be on line 98 but it's up to you

Reply:
All code has been formatted according to the kotlin code conventions